build.gradle

```diff
@@ -103,6 +103,9 @@ rat {
             'node_modules/**',
             '.idea/**',
             'README.md',
+            'docker/Dockerfile',
+            'docker/docker-compose.yml',
+            'docker/assets/',
```

Comment:
In general it's good practice, especially in public projects, to create Docker images with an easy to read Dockerfile. Also, if we do decide to publish images to the Dockerhub, we should use a Dockerfile in order to have the hub build the image and publish the dockerfile. This validates the image was built with the Dockerfile.

The docker-compose file will launch a holdmail instance with a MySQL backend for POC purposes. For the sake of ease of use we probably shouldn't add additional layers on top of it.